lib/configurators/aws/aws_test.go

```diff
@@ -1388,16 +1385,15 @@ func TestAWSDocumentConfigurator(t *testing.T) {
 	require.NoError(t, config.ApplyFileConfig(fileConfig, serviceConfig))
 
 	config := ConfiguratorConfig{
-		AWSSession:   &awssession.Session{},
-		AWSIAMClient: &iamMock{},
-		AWSSTSClient: &STSMock{ARN: "arn:aws:iam::1234567:role/example-role"},
-		AWSSSMClients: map[string]ssmiface.SSMAPI{
-			"eu-central-1": &SSMMock{
+		iamClient: &iamMock{},
+		stsClient: &stsMock{ARN: "arn:aws:iam::1234567:role/example-role"},
```

Comment:
nit: IMHO, ideally tests should mock an identity instead of `stsClient`. otherwise every place calls `GetIdentityWithClientV2` will need to mock sts, which is kinda indirect. but i think we do this all the places anyway.

Reply:
I just converted what was here already, but I think mocking sts makes the test closer to reality than mocking identity directly